modules/standard/IO.chpl

```diff
@@ -6348,6 +6349,35 @@ proc readf(fmt:string):bool {
 }
 
 
+/*
+   Skip a field in the current aggregate format. This method is currently only
+   supported for JSON format and returns ENOTSUP for other formats. In other
+   formats, it may not be possible in general to know when a field ends.
+
+   :arg error: optional argument to capture an error code. If this argument
+               is not provided and an error is encountered, this function
+               will halt with an error message.
+ */
+proc channel.skipField(out error:syserr) {
```

Comment:
This function supports the read-and-skip-unknown-fields. Suppose we had a record with a, b.
This function will skip c in this:
{"a":1, "b":2, "c":3}

But c isn't a field in our record - we don't know the name of the field we are skipping.
It skips the next field to be read. It does not include the field name.